jobs/kpi-forecasting/kpi_forecasting/results_processing.py

```diff
@@ -74,12 +74,13 @@ def _set_intra_forecast_agg_functions(self):
 
     def _load_config_data(self):
         """Extracts data from the list of config files passed to the class and stores it in the
-        config_data attribute. The filename is the key, and the contents (represnted as a DotMap)
+        config_data attribute. The filename is the key, and the contents
         are the values"""
         self.config_data = {}
         for config_file in self.input_config_list:
             full_path = f"{self.input_config_path}/{config_file}"
-            config_data = YAML(full_path).data
+            config_data = load_yaml(full_path)
+            print(config_data)
```

Comment:
Is this print statement leftover from debugging? Or do we want to print that to, e.g. the Airflow logs?

Reply:
whoops that should be taken out, fixed